mne/viz/backends/_pyvista.py

```diff
@@ -106,6 +106,61 @@ def visible(self, state):
         self.pts.SetVisibility(state)
 
 
+class _WebInteractor(object):
+    def __init__(self, renderer):
+        from IPython import display
+        from ipywidgets import HBox
+        self.renderer = renderer
+        self.plotter = self.renderer.plotter
+        self.fig, self.dh = self.screenshot()
+        self.controllers = self.configure_controllers()
+        layout = HBox([self.fig.canvas, self.controllers])
+        display.display(layout)
+
+    def screenshot(self):
+        import matplotlib.pyplot as plt
+        plt.ioff()
```

Comment:
Since this acts globally, we should probably try to save the state before changing it, and then resetting it when don